doc/Writerside/topics/first_data_flow.md

```
@@ -0,0 +1,28 @@
+# Добавляем первый поток данных
+
+После установки и первоначальной настройки подсистемы, мы готовы к добавлению первого потока данных.
+
+## 
+
+1. 
+
+Make sure that:
+- First prerequisite
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)